modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc

```diff
@@ -0,0 +1,438 @@
+
+|===
++
+.`sts_installer_privatelink_permission_boundary_policy.json`
+[%collapsible]
+====
+[source,json]
+----
+{
+"Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:ModifyVpcEndpointServiceConfiguration",
+        "route53:ListHostedZonesByVPC",
+        "route53:CreateVPCAssociationAuthorization",
+        "route53:AssociateVPCWithHostedZone",
+        "route53:DeleteVPCAssociationAuthorization",
+        "route53:DisassociateVPCFromHostedZone",
+        "route53:ChangeResourceRecordSets"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
+----
+====
++
+.ROSA installer vpc permission boundary policy, and policy files
+[cols="1,2",options="header"]
+|===
+|Policy|Description
+
+|`ManagedOpenShift-VPC-Permissions-Boundary-Policy``
+|Apply the IAM policy as a permission boundary on a role and it provides the ROSA installer with the permissions required to complete cluster installation tasks.
+
+|===
++
+.`sts_installer_vpc_permission_boundary_policy.json`
+[%collapsible]
+====
+[source,json]
+----
+{
+     "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+		    "ec2:AssociateDhcpOptions",
+		    "ec2:AssociateRouteTable",
+		    "ec2:AttachInternetGateway",
+		    "ec2:CreateDhcpOptions",
+		    "ec2:CreateInternetGateway",
+		    "ec2:CreateNatGateway",
+		    "ec2:CreateRouteTable",
+		    "ec2:CreateSubnet",
+		    "ec2:CreateVpc",
+		    "ec2:DeleteDhcpOptions",
+		    "ec2:DeleteInternetGateway",
+		    "ec2:DeleteNatGateway",
+		    "ec2:DeleteRouteTable",
+		    "ec2:DeleteSubnet",
+		    "ec2:DeleteVpc",
+		    "ec2:DetachInternetGateway",
+		    "ec2:DisassociateRouteTable",
+		    "ec2:ModifySubnetAttribute",
+		    "ec2:ModifyVpcAttribute",
+		    "ec2:ReplaceRouteTableAssociation"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+----
+====
++
+[NOTE]
+====
+Policies pass their json file as a permission boundary in the ROSA (`rosa`) CLI where the installer policy contains a custom boundary.
+====
++
+<3> Optional: Specify a custom ARN path for your account-wide roles. The path must contain alphanumeric characters only and start and end with `/`, for example `/test/path/dev/`. For more information about ARN paths, see link:https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html[Amazon Resource Names (ARNs)] in the AWS documentation.
+<4> Select the role creation mode. In `manual` mode, the ROSA CLI generates the `aws` commands needed to create the roles and policies. In `manual` mode, the corresponding policy JSON files are also saved to the current directory. `manual` mode enables you to review the details before running the `aws` commands manually. `Auto` mode automatically creates the OpenShift Cluster Manager IAM role and links it to your Red Hat organization account.
```

Comment:
Should `Auto` be `auto`?

For "OpenShift Cluster Manager", I think you can use an attribute.

"Red Hat" should be "Red{nbsp}Hat". See https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces.